frontend/plugins/frontline_ui/src/modules/integrations/facebook/components/FacebookPostTrigger.tsx

```
@@ -0,0 +1,63 @@
+      <Popover.Trigger asChild>
+        <Button
+          disabled={loading}
+          variant="ghost"
+          className="font-normal text-muted-foreground"
+        >
+          {loading ? (
+            <Skeleton className="h-4 w-16" />
+          ) : (
+            <span
+              className="flex-auto max-w-32 truncate"
+              dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)